SimpleTrustZone/SimpleTZ.csolution.yml

```diff
@@ -4,10 +4,12 @@ solution:
   compiler: AC6
 
   packs:
-    - pack: ARM::CMSIS@5.9.0
+    - pack: ARM::CMSIS@6.0.0
```

Comment:
Do we need to pin version?

Reply:
we don't really need to pin to a certain version, that line of code was set to CMSIS@5.9.0, so I just re-pinned it. Will change it to a floating version then